### PR TITLE
feat: add text input component styles

### PR DESCRIPTION
**Pull request by olehrakwix**
image

image

### CODE REVIEW

File: _codux/ui-kit/components/components.board.tsx [hunk 1]
```diff
@@ -17,6 +17,25 @@ export default createBoard({
         <ComponentWrapper>
             <Kit category="Core Components" title="Components & Elements">
                 <Kit.Section title="Inputs">
+                    <Kit.Item>
+                        <Variant name="Input">
+                            <input className="text-input" value="text input" onChange={() => {}} />
```
Comment (yurii-ve) [inline _codux/ui-kit/components/components.board.tsx:22]: I'm thinking if we should provide `value` and `onChangeProps` as it means that user won't be able to type anything in the board.
On the other hand these are convenient to have when user drops this input from the add panel.

@alisey what do you think?

File: src/styles/common.scss [hunk 2]
```diff
@@ -90,6 +90,28 @@
...
+    background: transparent;
+    padding: 8px 0;
+    border-bottom: solid 1px rgba(23, 17, 13, 0.4);
+}
+
+input.text-input {
+    box-shadow: none;
+}
+
+input.text-input:disabled {
```
Comment (yurii-ve) [inline src/styles/common.scss:106]: how does the placeholder text look like? maybe it should be styled it as well?

Comment (olehrakwix) [reply src/styles/common.scss:106]: in wix template there is no placeholders:
Screenshot 2024-11-05 at 12 56 33

so I'm not sure what is expected in this case

Comment (olehrakwix) [reply src/styles/common.scss]: anyway, added some styles for placeholder and added this state to a board

**Review by yurii-ve — APPROVED** [review]

File: src/styles/common.scss [hunk 3]
```diff
@@ -113,6 +113,29 @@
     color: inherit;
 }
 
+input.text-input {
```
Comment (alisey) [inline src/styles/common.scss:116]: Don't use kebab case.

**Pull request merged**